tests/framework/behaviors/SluggableBehaviorTest.php

```diff
@@ -229,4 +245,20 @@ public function behaviors()
             ],
         ];
     }
-}
\ No newline at end of file
+}
+
+class ActiveRecordSluggableSkipOnEmpty extends ActiveRecordSluggable
+{
```

Comment:
the naming should have ActiveRecord at the end (ActiveRecordSluggable actually too IMO). So a better name is `SkipOnEmptySluggableActiveRecord'. Perhabs you can change that too.